concepts/comparison/about.md

```diff
@@ -0,0 +1 @@
+# About
```

Comment:
You will have thought of this yourselves, but I do want to point out that it might be useful to add a TODO in the text. This will make it clear to both maintainers that something needs to be done, makes it easy to find what needs to be done and makes the display on the website less weird (I'd prefer an empty TODO over an empty document)